tests/checkpointing/test_model_checkpoint.py

```diff
@@ -1217,8 +1217,8 @@ def make_assertions(cb_restore, written_ckpt):
             "kth_best_model_path": False,
             "kth_value": False,
             "best_k_models": False,
+            "last_model_path": False,
```

Comment:
Can you add a test? You can mimic the behaviour shown by Adrian in https://github.com/PyTorchLightning/pytorch-lightning/pull/12045#issuecomment-1051290390

Reply:
Yes, that makes sense. Done! Please let me know if it looks good. :)